src/docs/product/issues/issue-details/performance-issues/large-http-payload/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Large HTTP Payload"
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/large-http-payload/
+description: "Learn more about Large HTTP Payload issues, how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Large HTTP payload issues are created when an http span has an encoded body size that exceeds a threhsold.
+
+## Detection Criteria
+
+The detector for this performance issue looks for a single http span that meets the following criteria
+
+- The http method must be a GET, POST, DELETE, PUT, PATCH
+- The http url must not begin with _next/static/ or _next/data/
+- The http url must not end with the extensions .js, .css, .svg, .png, .mp3, .jpg, .jpeg
+
+Once an http span that meets this criteria is found, the encoded body size is checked against a threshold.
+
+If Sentry isn't detecting a large http payload issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
```

Comment:
```suggestion
## Detection Criteria

The detector for this performance issue looks for a single http span that meets the following criteria:

- The http method must be a GET, POST, DELETE, PUT, PATCH
- The http url must not begin with _next/static/ or _next/data/
- The http url must not end with the extensions .js, .css, .svg, .png, .mp3, .jpg, .jpeg

Once an http span that meets this criteria is found, the encoded body size is checked against a threshold. If Sentry isn't detecting a large http payload issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
```